ampcrowd/crowd_server/settings.py

```diff
@@ -129,7 +138,7 @@
         'NAME': 'sampleclean',
```

Comment:
Could we change these to `'ampcrowd'`? Sampleclean is an artifact of my other project and shouldn't be relevant here.

Installation instructions should also be updated accordingly.